### ISSUE TITLE
Can't add the "tool" tag on a methods page 

### ISSUE THREAD

**Issue by steviepubliclab** [publiclab/plots2#1544]
### What happened just before the problem occurred
I was working on the new /wiki/microscope page and when I went to add the tag "tool" I got this error. 

![screen shot 2017-07-18 at 11 53 23 am](https://user-images.githubusercontent.com/7490478/28329801-c59e4ab0-6bb0-11e7-9a50-682b9c189a1e.png)

**Comment by jywarren** [comment]
I went into the logs and found the error is:

```
Started GET "/tag/create/14696?remote=true&name=tool&_=1501273696770" for 66.30.8.89 at 2017-07-28 20:30:21 +0000
Processing by TagController#create as JS
  Parameters: {"remote"=>"true", "name"=>"tool", "_"=>"1501273696770", "nid"=>"14696"}
Completed 404 Not Found in 52.1ms

ActiveRecord::RecordNotFound (Couldn't find User without an ID):
  app/models/tag.rb:203:in `followers_who_dont_follow_tags'
  app/mailers/subscription_mailer.rb:32:in `notify_tag_added'
  app/models/node.rb:669:in `block in add_tag'
  app/models/node.rb:653:in `add_tag'
  app/controllers/tag_controller.rb:133:in `block in create'
  app/controllers/tag_controller.rb:127:in `each'
  app/controllers/tag_controller.rb:127:in `create'
```

That's on this line! https://github.com/publiclab/plots2/blob/master/app/models/tag.rb#L203

**Comment by jywarren** [comment]
Fixed in #1556, thanks!
